longevity_tombstone_gc_test.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import time
+from functools import partial
+
+from longevity_twcs_test import TWCSLongevityTest
+from sdcm.utils.common import ParallelObject
+from sdcm.utils.sstable_utils import SstableUtils
+
+
+class TombstoneGcLongevityTest(TWCSLongevityTest):
+    keyspace = 'scylla_bench'
+    table = 'test'
+    ks_cf = f'{keyspace}.{table}'
+    propagation_delay = 60 * 4  # setting a value shorter than the default, of 4 minutes
+    repair_date = None
+    db_node = None
+
+    def _run_repair_and_major_compaction(self, wait_propagation_delay: bool = False):
+        self.log.info('Run a flush for user-table on nodes')
+        triggers = [partial(node.run_nodetool, sub_cmd=f"flush -- {self.keyspace}", ) for node in self.db_cluster.nodes]
+        ParallelObject(objects=triggers, timeout=1200).call_objects()
+        # for node in self.db_cluster.nodes:
+        #     node.run_nodetool(f"flush -- {self.keyspace}")
+
+        self.log.info('Run a repair for user-table on nodes')
```

Comment:
same here, it should say in which keyspace and table we will run the repair, and in this case, seems to be `f'{self.keyspace}.{self.table}'`

Reply:
fixed